src/blockchain.js

```diff
@@ -59,19 +66,29 @@ class Transaction{
 }
 
 class Block {
+    /**
+     * @param {Date} timestamp 
+     * @param {Transaction[]} transactions 
+     * @param {string} previousHash 
+     */
     constructor(timestamp, transactions, previousHash = '') {
         this.previousHash = previousHash;
-        this.timestamp = timestamp;
+        this.timestamp = timestamp.toISOString();
```

Comment:
I see no need for us to convert it to an ISO string. Better to keep it as a number so we can sort easily (for instance).

Reply:
I would pass in a `Date` because i think it is more usable (and more understandable)

Reply:
Actually a number is more usable because you can more easily sort it.
So for now: revert back to the number and maybe you can open a second PR and create a method to parse the data to an ISO string?

Reply:
After this change we're ready to merge it in!

Reply:
ok, can you shortly say me where this sort is?

Reply:
reverted in commit be70fd3

Reply:
There is no sort yet, but that's the way other cryptocurrencies store the data. They don't store a readable format, but instead a format that still has to be parsed.